src/ScheduledTask/ProcessNotificationsHandler.php

```diff
@@ -24,25 +24,41 @@
 
 namespace Adyen\Shopware\ScheduledTask;
 
-use Adyen\Shopware\AdyenPaymentShopware6;
 use Adyen\Shopware\Entity\Notification\NotificationEntity;
-use Adyen\Shopware\NotificationProcessor\NotificationProcessorFactory;
+use Adyen\Shopware\Entity\Refund\RefundEntity;
+use Adyen\Shopware\Provider\AdyenPluginProvider;
```

Comment:
Looks like the `AdyenPluginProvider` is no longer used in this handler. Or is it?

Reply:
Great catch! Updated in 38c0b13